docs/source/conf.py

```diff
@@ -235,7 +235,7 @@ def get_authors_from_citation_file(file_name):
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
+intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
```

Comment:
This really makes a difference?

Reply:
This wasn't done by me but by `black` in the pre-commit (see commit message in [ad1769c](https://github.com/gammasim/simtools/pull/478/commits/ad1769cbb3048164e7e44b0e352b6a49cf6c4057)). This is why I said I think you didn't use the pre-commit hook.